src/ScClient.ts

```diff
@@ -183,9 +183,18 @@ export class ScClient {
     });
   }
 
-  public async createElementsBySCs(scsText: string[]) {
+  public async createElementsBySCs(scsText: string[] | ISCs[]) {
     return new Promise<boolean[]>((resolve, reject) => {
-      this.sendMessage("create_elements_by_scs", scsText, ({ payload, errors }) => {
+      const payload = scsText.map((scsString) => {
+        if(typeof scsString === "string")
+        {
+          return {scs: scsString, outputStructure: 0 }
+        }
+        else{
+          return {scs: scsString.scs, outputStructure: scsString.outputStructure?.value as number}
```

Comment:
If scsString.outputStructure is null client send this null to sc-server. Client must send 0, not null